src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import { Component } from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// APP
+/* eslint-disable */
```

Comment:
- [ ] Good job converting this to a class-based component as suggested by the previous reviewer, but i can see you disabled this linter. Kindly note that linters force you to write clean and good code, so please make sure you fix your linter issues rather than disabling them 😄 

for this linter issue, I will suggest you add a constructor and an empty state. This way, the error will disappear as you have converted the component to a stateful component 👍🏽 


e.g 

```
  constructor(props) {
    super(props);
    this.state = {};
  }
```

Reply:
**@vickymarz** It really solves the issue 🤩, thank you very much 👏👏👏